Add pages for latexmk and pdflatex

85pando opened the pull request:
Pdflatex and latexmk are ywo tools to create Output documents from LaTeX source files.

waldyrious commented on pages/common/latexmk.md:
```diff
@@ -0,0 +1,24 @@
+# latexmk
+
+> Compile LaTeX source files into finished documents. Does multiple runs when
+> the output will change.
```
1) Put each sentence into its own line
2) The second sentence isn't clear. Can you rephrase it?

waldyrious commented on pages/common/latexmk.md:
```diff
@@ -0,0 +1,24 @@
+> Compile LaTeX source files into finished documents. Does multiple runs when
+> the output will change.
+
+- Compile a dvi document.
```
Spell out what "dvi" stands for, in parenthesis

waldyrious commented on pages/common/latexmk.md:
```diff
@@ -0,0 +1,24 @@
+
+`latexmk -pdf {{source.tex}}`
+
+- Clean up temporary tex files created for a single tex file.
```
suggestion: single --> specific

85pando commented:
* Tried to rephrase the sentence about multiple runs.
* Elongated dvi in parentheses.
* reordered commands.

    * Should I put the `-C` flag also before? I ordered it like this, because in my experience it's not that often that you want to delete your outputfiles.

waldyrious commented on pages/common/pdflatex.md:
```diff
@@ -0,0 +1,11 @@
+
+> Compile pdf document from LaTeX source files.
+
+- Compile pdf document:
```
"Compile *a* pdf document" (as in the other command)

waldyrious commented on pages/common/pdflatex.md:
```diff
@@ -0,0 +1,11 @@
+
+`pdflatex {{source.tex}}`
+
+- Compile pdf document, halt on each error:
```
"Compile **a** pdf document, halt**ing** on each error"

waldyrious commented:
Thanks for the improvements, much better now :+1: 

> Should I put the -C flag also before?

No, I think you made a good call :)

Before I can endorse this, could you address the inline comments I left? Also the one about "single" --> "specific", if it was an overlook -- otherwise please explain why you disagree.

85pando commented:
* Changed the descriptions of the pdf version as suggested.
* Added an additional command with no options (did not know it also does this ☻ )

For the "single" --> "specific" comment, I understood it as a comment on my ordering of the commands of `latexmk -c {{source.tex}}` coming before `latexmk -c` which I then changed.
If adding the command without options was what you meant, all is fine now, otherwise please advise.

waldyrious commented:
Sorry if I wasn't clear. I meant changing the description from "Clean up temporary tex files created for a **single** tex file:" to "Clean up temporary tex files created for a **specific** tex file:" (although "single" works, I believe "specific" gets the idea across better)

85pando commented:
Allgright, got you now ☺

This should be it then.

waldyrious commented:
Looks great! :+1:

rubenvereecken commented:
Live and learn, thanks a lot @85pand, I could actually use this one :+1:

85pando commented:
Isn't that the essence of tldr? ☻
